### PR TITLE
Add steps to apply secrets in the README

### PR DESCRIPTION
**Pull request by akshmish**
**What this PR does / why we need it**:
1. Adding an additional step to apply the secrets as it may not be obvious to every reader
2. Update the NDB public documentation link from NDB 2.4 -> NDB 2.5 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
N/A

**How Has This Been Tested?**:
N/A

**Special notes for your reviewer**:
Verify the README page looks fine after the PR has been merged

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ make deploy
 
 ### Using the Operator
 
-1. Create the secrets that are to be used by the custom resource:
+1. Create a secrets.yaml that will store the secrets to be used by the custom resource:
+You may choose to create this file under /samples dir
```
Comment (manavrajvanshi) [inline README.md:33]: Can we remove this line since we're already mentioning `<path/to/secrets.yaml>` in the next instruction.

File: config/samples/ndb_v1alpha1_database.yaml [hunk 2]
```diff
@@ -4,12 +4,12 @@ metadata:
...
 spec:
   ndb:
-    clusterId: "Nutanix Cluster Id"
+    clusterId: [Nutanix Cluster Id]
     credentialSecret : ndb-secret-name
     server: https://[NDB IP]:8443/era/v0.9
     skipCertificateVerification: true
   databaseInstance:
-    databaseInstanceName: "Database Instance Name"
+    databaseInstanceName: [Database Instance Name]
```
Comment (manavrajvanshi) [inline config/samples/ndb_v1alpha1_database.yaml:12]: Enclosing these values in '[ ..... ]' will change the types `clusterId` and `databaseInstanceName` to a list type. I don't think that's required for these fields now, can we revert these changes?

**Review by manavrajvanshi — CHANGES_REQUESTED** [review]
Minor comments, please revert the unintended '[...]'.

**Comment by manavrajvanshi** [comment]
@akshmish Since you're already updating the README.md, can you also add the change to update the ndb user guide in the file?
https://portal.nutanix.com/page/documents/details?targetId=Nutanix-NDB-User-Guide-v2_5:Nutanix-NDB-User-Guide-v2_5

File: config/samples/ndb_v1alpha1_database.yaml (hunk 2 above)
Comment (akshmish) [reply config/samples/ndb_v1alpha1_database.yaml:12]: The original plan was to keep one single pattern for all the values "that need to be updated". 

as in  "server: https://[NDB IP]:8443/era/v0.9"

But I am ok to revert.

**Comment by akshmish** [comment]
> @akshmish Since you're already updating the README.md, can you also add the change to update the ndb user guide in the file? https://portal.nutanix.com/page/documents/details?targetId=Nutanix-NDB-User-Guide-v2_5:Nutanix-NDB-User-Guide-v2_5

Done

File: README.md [hunk 3]
```diff
@@ -47,20 +48,33 @@ stringData:
...
+kubectl apply -f <path/to/secrets.yaml>
+
+```
+You can optionally verify that they have been created:
+
+```sh
+kubectl get secrets
+```
+
+2. To create an instances of CR (provision databases), update the NDB Server IP in [ndb_v1alpha1_database.yaml](config/samples/ndb_v1alpha1_database.yaml) in spec section. Ensure that the names of the secrets provided in the ndb_v1alpha1_database.yaml exactly match with the names of secrets defined in the secrets.yaml
```
Comment (manavrajvanshi) [inline README.md:71]: Since we're getting verbose in this point, can you include the clusterId and other details too?

**Review by manavrajvanshi — DISMISSED** [review]

**Review by shenoypritika — APPROVED** [review]

**Review by manavrajvanshi — APPROVED** [review]
